exercises/affine-cipher/tests/affine-cipher.rs

```diff
@@ -0,0 +1,146 @@
+use affine_cipher::*;
+
+#[test]
+fn encode_yes() {
+    assert_eq!(encode("yes", 5, 7).unwrap(), "xbt")
```

Comment:
There was once upon a time in https://github.com/exercism/rust/issues/338 where we thought all assertions of the form `assert_eq!(a.unwrap(), b)` should instead be written as `assert_eq(a, Ok(b))` because it gave better test outputs when the implementation said `Err` when it should instead give `Ok`.

However, I have just given this a try, and here is a comparison of the two:

```
---- tests::dont_unwrap_and_expect_ok_string stdout ----
thread 'tests::dont_unwrap_and_expect_ok_string' panicked at 'assertion failed: `(left == right)`
  left: `Err(NotCoprime(2))`,
 right: `Ok("hello")`', src/lib.rs:16:9

---- tests::do_unwrap_and_expect_string stdout ----
thread 'tests::do_unwrap_and_expect_string' panicked at 'called `Result::unwrap()` on an `Err` value: NotCoprime(2)', src/libcore/result.rs:1084:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

I deem this sufficiently helpful (shows what was attempted to be unwrapped) that we no longer need to consider https://github.com/exercism/rust/issues/338. In fact this has always been the case (it had always shown the `Err` that was `unwrap`ped, even at the time of filing https://github.com/exercism/rust/issues/338), so it's simply that this track has generally gotten better at having good `Error`s, and this caused me to change my mind on the issue.

I think the former is still slightly more helpful (shows what was actually expected) and should be used in situations where it is possible without inconvenience, but note that since this is a `Result<String, AffineCipherError>`, we don't seem to just be able to write `Ok("xbt")`, but instead would have to write `Ok("xbt".to_string())` or something. I deem that enough of an inconvenience so as to say it need not happen here. I leave it as a comment in case anyone feels differently.